pkg/deploy/deploy.go

```diff
@@ -130,7 +129,7 @@ func DownloadManifests(uri string) error {
 	return nil
 }
 
-func DeployManifestsFromPath(owner metav1.Object, cli client.Client, componentName, manifestPath, namespace string, s *runtime.Scheme, componentEnabled bool) error {
+func DeployManifestsFromPath(cli client.Client, owner metav1.Object, manifestPath, namespace, componentName string, componentEnabled bool) error {
```

Comment:
```suggestion
func DeployManifestsFromPath(cli client.Client, owner metav1.Object, manifestPath string, namespace string, componentName string, componentEnabled bool) error {
```

Reply:
I can do that, but wondering if that's a rule for this project?

Reply:
dont think we have any defined design guide or coding rule.
just feel if we are doing this everywhere wihtin the repo, could be good to have consistency. 
could be i missed something. free to make your call.

Reply:
I think I did an accidental cleanup, so I'm happy to roll it back. And I'm all for consistency. On that note - what do you think about introducing a linter? We've successfully used it in other projects. Makes life easier and allows you to focus on important things while doing a PR review. It can be a PITA when introduced to an already existing project but I can take care of it.

Reply:
linter is a good idea. a while ago i checked other repo in ODH, some had it as part of the GH action, some did not. 
i did not follow that up .....
if you have something handy, free to make a PR, happy to get code in a better shape :D